### PR TITLE
mtls: Add MTLS for use with admin ingresses

### PR DESCRIPTION
**Pull request by drGrove**

### CODE REVIEW

File: mtls/service.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mtls
+  labels:
+    app.kubernetes.io/name: mtls
+spec:
+  type: ClusterIP
```
Comment (daurnimator) [inline mtls/service.yaml:8]: this is the default; avoid specifying

File: mtls/service.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  ports:
+    - port: 4000
+      targetPort: http
+      protocol: TCP
+      name: http
+  selector:
+    app.kubernetes.io/name: mtls
+
```
Comment (daurnimator) [inline mtls/service.yaml:16]: empty line

File: mtls/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+configMapGenerator:
+  - name: mtls
+    behavior: replace
+    files:
+      - files/config.ini
+  - name: mtls-admin-seeds
+    behavior: replace
+    files:
+      - files/admin_seeds/dannygrove.asc
+patchesJson6902:
```
Comment (daurnimator) [inline mtls/kustomization.yaml:20]: could you move this up next to `patches:`?

File: mtls/files/admin_seeds/lrvick.asc [hunk 4]
```diff
@@ -0,0 +1,2331 @@
...
+mQINBEoE5H4BEACwigcH2W+tLHRC++V0FD6ppThtZCEBPo4Kf595N4dkOCRzqZrn
+uZeokptTFdb2Xcsi4kQXfx2gTsQStri5P8nFywG8dTniful3BNuLbD9WuNFhWkg2
+TZx64BsJEzBOBeKluLsiN0IpgAj1KA9PGBNU7U0DEGEsw8HVGDcUVYhg/gVcWQVG
+O+Z1UYLpdws9lmMnTwg8KVudjL3YcUB/ybsLat4qfhsH5hjFsXQwwlaNh3GlDpkc
+qldoZ2WjYEtw/U9rZadc3HPYvRp4keyb7aGfdGDgDBqXGbtjdHvM5sqiQHgWaw9E
+RcUrICsZX6vS8+lLx+TLjzX1/bwNgtOVmiBnnWmcB+JDtTNWI5ux+6F2vq65IXXV
+1m0kPZo6B0w0yYOayOKvgMXtQ0hNuoRZO41tO2W6P114FwTQIKNwrUh6Ypb1ty6E
+Sp/hHDPUopcSezpt1oS0IHDOKVKoUBRkN7rne+bNUtQ+O9hEyZlxjbFnSpa6KzjL
+A4TB/3qm6H8pBgbzqyQQQOrXK8qFOpRYk6lOXdrFwFhXO28GLQFkNk3wWnBrTp0t
+OBj7RA5wL8BTEWSwz9aa2dU/SdwExRxxbzweMbJpsaDdtu06njLUXNQvA8aMiVOI
```
Comment (daurnimator) [inline mtls/files/admin_seeds/lrvick.asc:12]: theses keys are quit long; can you remove some packets?

File: mtls/kustomization.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  - secret-generator.yaml
+configMapGenerator:
+  - name: mtls
+    behavior: replace
+    files:
+      - files/config.ini
+  - name: mtls-admin-seeds
+    behavior: replace
+    files:
+      - files/admin_seeds/dannygrove.asc
```
Comment (daurnimator) [inline mtls/kustomization.yaml:19]: missing the other admins?

File: mtls/files/admin_seeds/lrvick.asc (hunk 4 above)
Comment (drGrove) [reply mtls/files/admin_seeds/lrvick.asc:12]: As soon as the refresh kicks it's going to pull in the updated keys anyways. So we could just strip it down to a single UID

**Pull request merged**